developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -134,25 +140,40 @@ import QuickStartCode from '!!raw-loader!/_includes/code/graphql.filters.nearTex
     language="py"
   />
 
+Note that the `Move` import exposes configuration elements for moving `near_text` search towards, or away from, the main query input.
+
 </TabItem>
 </Tabs>
 
-The `wvc` import exposes basic types that you will need for most applications.
-
-```python
-import weaviate.classes as wvc
-```
+The `wvc` namespace exposes commonly used classes that you will need for most applications, [divided further into submodules based on their primary purpose](./index.md#helper-classes).
 
-The `Move` import exposes configuration elements for `near_text` search. If your application doesn't need `near_text` search. Your can streamline your code namespace by omitting this import.
+<FilteredTextBlock
+  text={PythonCode}
+  startMarker="# START WVCImportExample"
+  endMarker="# END WVCImportExample"
+  language="py"
+/>
 
-```python
-from weaviate.collections.classes.grpc import Move
-```
 ### Interact with collections
 
-When you connect to a Weaviate database, the v4 client and the v3 client return different objects. The v3 client returns a `client` object that you interact with for CRUD and search operations. The v4 client uses a `collection` object instead.
+When you connect to a Weaviate database, the v4 API returns a `WeaviateClient` object, and the v3 API returns a `Client` object.
+
+The `v3` API's interactions were built around the `client` object (an instance of `Client`), including server interactions for CRUD and search operations.
+
+With the `v4` API, the main starting points for your interaction with Weaviate follow a different paradigm.
```

Comment:
Consider:
In the `v4` API, the main starting points for your interaction with Weaviate follow a different paradigm.